web/Weblate-server.qmd

```diff
@@ -205,26 +206,68 @@ rebase refs/remotes/origin/trunk: command returned error: 1
 Weblate will provide a suggestion on how to fix, but in short:
 
 1. SSH to the Weblate server
-2. Attach the Docker container:
+2. Make a backup on the Git repo before messing with the rebase process:
+
+    ```sh
+    tar -zcvf /root/base-r-gui-repo-$(date '+%Y-%m-%d').tar.gz \
+      /var/lib/docker/volumes/weblate-docker_weblate-data/_data/vcs/r-project/base-r-gui
+    ```
+
+3. Attach the Docker container:
 
     ```sh
     sudo docker exec -ti weblate-docker_weblate_1 bash
     ```
 
-3. Enter the git folder:
+4. Enter the git folder:
```

Comment:
I would usually write the numbering as:

```
1. first item
1. second item
1. third item
```

Then markdown handles the numbering. Saves from having to re-number items like here when something gets added. I'm just not sure if `.qmd` will support it properly, any idea?

Reply:
GH preview of the edit with all `1.` didn't show anything, I think we have to check on the output wiki whether it translates. Let's test that out in a follow-up.

Reply:
#24